contracts/token/ERC721/ERC721.sol

```diff
@@ -19,8 +26,8 @@ contract ERC721Enumerable is ERC721Basic {
  * @dev See https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md
  */
 contract ERC721Metadata is ERC721Basic {
-  function name() public view returns (string _name);
-  function symbol() public view returns (string _symbol);
+  function name() external view returns (string _name);
+  function symbol() external view returns (string _symbol);
   function tokenURI(uint256 _tokenId) public view returns (string);
```

Comment:
This is not part of the PR, but it made me notice `tokenURI` is supposed to be `external` as per the spec: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md

Is this something we want to fix before merging?

Reply:
Given that public implies external, I wouldn't bother in making this change now, but rather wait until it is implemented in OZ and then merged here